src/kaltura-player.js

```diff
@@ -173,9 +195,12 @@ class KalturaPlayer extends FakeEventTarget {
    */
   configure(config: Object = {}): void {
     config = supportLegacyOptions(config);
-    const configDoctionary = Utils.Object.mergeDeep({}, this.config, config);
-    evaluatePluginsConfig(config.plugins, configDoctionary);
-    this._localPlayer.configure(config);
+    const configDictionary = Utils.Object.mergeDeep({}, this.config, config);
+    evaluatePluginsConfig(config.plugins, configDictionary);
+    const localPlayerConfig = Utils.Object.mergeDeep({}, config);
+    delete localPlayerConfig.plugins;
```

Comment:
why does it matter if you pass the plugins or not?
We are already passing the ui config which we don't need.
We can extract the config and pass the relevant config but I'm not sure if it's needed

Reply:
the plugins are not relevant for playkit. I would remove the UI as well

Reply:
Let's make a helper function for that, you'll probably do the same here and in the constructor.

Reply:
@yairans you didn't remove the irrelevant config